crates/fluvio-protocol/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "fluvio-protocol"
 edition = "2021"
-version = "0.10.11"
+version = "0.11.0"
```

Comment:
Since all that was changed is some added error codes, let's just bump the patch level on this crate instead of the minor for now

Reply:
Fixed!

Reply:
Fixed!